### PR TITLE
Magically increase code coverage

### PR DESCRIPTION
**Pull request by Stebalien**

### CODE REVIEW

**Comment by Stebalien** [comment]
Compiled:

```json
{
  "comment": false,
  "ignore": [
    "(?s:.*_gen\\.go\\$)\\Z",
    "(?s:.*/mocks_full\\.go\\$)\\Z",
    "^chain/actors/builtin/[^/]*/(message|state|v)[0-4]\\.go$",
    "(?s:api/test/.*)\\Z",
    "(?s:conformance/.*)\\Z",
    "(?s:gen/.*)\\Z",
    "(?s:chain/actors/agen/.*)\\Z",
    "(?s:cmd/lotus\\-shed/.*)\\Z",
    "(?s:cmd/tvx/.*)\\Z",
    "(?s:cmd/lotus\\-pcr/.*)\\Z",
    "(?s:cmd/tvx/.*)\\Z",
    "(?s:cmd/lotus\\-health/.*)\\Z",
    "(?s:cmd/lotus\\-fountain/.*)\\Z",
    "(?s:cmd/lotus\\-stats/.*)\\Z",
    "(?s:cmd/lotus\\-pcr/.*)\\Z",
    "(?s:cmd/lotus\\-sim/.*)\\Z",
    "(?s:cmd/lotus\\-bench/.*)\\Z",
    "(?s:cmd/lotus\\-seed/.*)\\Z",
    "(?s:cmd/chain\\-noise/.*)\\Z",
    "(?s:api/docgen/.*)\\Z",
    "(?s:api/docgen\\-openrpc/.*)\\Z"
  ],
  "github_checks": {
    "annotations": false
  }
}
```

Note: I don't think these apply until merged.

File: .codecov.yml [hunk 1]
```diff
@@ -1,19 +1,29 @@
...
-  - "cmd/lotus-chainwatch/*"
-  - "cmd/lotus-health/*"
-  - "cmd/lotus-fountain/*"
-  - "cmd/lotus-townhall/*"
-  - "cmd/lotus-stats/*"
-  - "cmd/lotus-pcr/*"
+  # Auto generated
+  - "**_gen.go$"
+  - "**/mocks_full.go$"
+  - "^chain/actors/builtin/[^/]*/(message|state|v)[0-4]\\.go$" # We test the latest version only.
```
Comment (ZenGround0) [inline .codecov.yml:6]: nit: it would be cool if this '4' was kept in a variable somewhere that got auto-updated by the actors release gen code.

**Review by ZenGround0 — APPROVED** [review]
LGTM

File: .codecov.yml (hunk 1 above)
Comment (Stebalien) [reply .codecov.yml:6]: That would be nice, but maintaining this file with code-gen sounds like a pain.

**Pull request merged**